### DISCUSSION TITLE
Msfvenom android payload

### DISCUSSION THREAD

**Discussion by PePsI850** [rapid7/metasploit-framework#19350] [Q&A]
Good afternoon Metasploit team. I'm a user of a Metasploit framework (which is the best exploitation framework :) ) and I have a issue with the android meterpreter payload ( payload/android/meterpreter/reverse_tcp ) this payload unfortunately has compatible issue with the latest android version ( 14 ) its not working on android 14 just works until 13. Please can you update/upgrade this payload compatibility to 14? Thank you.

**Comment by h00die** [comment]
#19154
